docs/UserGuide.md

```diff
@@ -2,22 +2,22 @@
 
 ## Table of Contents
 <pre>
-<a href =#intro>1. Introduction </a>
-<a href =#start>2. Quick Start </a>
-<a href =#init>3. Initialisation </a>
-<a href =#features>4. Features </a>      
+<a href =#intro>1 Introduction </a>
+<a href =#start>2 Quick Start </a>
+<a href =#init>3 Initialisation </a>
+<a href =#features>4 Features </a>      
    <a href =#subjects>4.1 Subjects </a>
    <a href =#topics>4.2 Topics </a>
    <a href =#tasks>4.3 Tasks </a>
    <a href =#flashcards>4.4 Flashcards </a>
    <a href =#takeQuiz>4.5 Taking Quizzes </a>
-   <a href =#result>4.6. Viewing quiz results </a>
-   <a href =#store>4.7. Storing data </a>
-   <a href =#export>4.8. Exporting data </a>
-<a href =#faq>5. FAQ </a>
-<a href =#summary>6. Command Summary </a>
+   <a href =#result>4.6 Viewing quiz results </a>
+   <a href =#store>4.7 Storing data </a>
+   <a href =#export>4.8 Exporting data </a>
+<a href =#faq>5 FAQ </a>
+<a href =#summary>6 Command Summary </a>
 </pre>
-## 1. Introduction <a name="intro"></a>
+## 1 Introduction <a name="intro"></a>
 
```

Comment:
For headings such as 1. Introduction, there should be a fullstop after the number iirc. For subheadings such as 4.1 Subjects they are not needed.